packages/create-config/index.js

```diff
@@ -11,7 +11,12 @@ const fileExists = filePath => {
 
 const getPackageManager = () => {
   // get the root of the repository
-  const repositoryRoot = execSync('git rev-parse --show-toplevel').toString().trim();
+  let repositoryRoot = '';
```

Comment:
I think the end result is the same but since we're using `path.join` below, I think it might make sense to use `process.cwd()` as the default value?

Reply:
I used the empty string because the code before #738 was simply `if (fileExists('bun.lockb'))`, but we could definitely use `process.cwd()` as the default value instead.